worlds/shivers/__init__.py

```diff
@@ -219,7 +219,8 @@ def fill_slot_data(self) -> dict:
             "ElevatorsStaySolved": self.options.elevators_stay_solved.value,
             "EarlyBeth": self.options.early_beth.value,
             "EarlyLightning": self.options.early_lightning.value,
-            "FrontDoorUsable": self.options.front_door_usable.value
+            "FrontDoorUsable": self.options.front_door_usable.value,
+            "PuzzleCollectBehavior": self.options.puzzle_collect_behavior.value
```

Comment:
```suggestion
            "PuzzleCollectBehavior": self.options.puzzle_collect_behavior.value,
```
If you put a comma at the end here, then if you add another thing to slot data later, it'll only show the new line as an addition, instead of also showing the previous line as changed

Reply:
Done.